common/deposits/hfe_deposit.txt

```diff
@@ -346,4 +367,211 @@ hfe_deposit_memorial_4_0 = {
 			has_deposit = hfe_deposit_memorial_4_0
 		}
 	}
-}
\ No newline at end of file
+}
+
+###################################################
+
+#	VARELSE		#
+
+hfe_deposit_mechanicus_farm_spire = {
+	icon = d_broken_spire
+	is_for_colonizeable = yes
+	category = deposit_cat_food
+	use_for_min_max_adjustments = no
+	use_weights_for_terraforming_swap_types = no
+	should_swap_deposit_on_terraforming = no
+
+	potential = {
+		planet = {
+			OR = {
+				has_planet_flag = hfe_colony_beta
+				has_planet_flag = hfe_varelse_colony
+				owner = { has_country_flag = hfe_varelse_primitive_0 }
+			}
+		}
+	}
+
+	resources = {
+		category = hfe_godforge
+		produces = {
```

Comment:
This 2 produces nodes should be same, primitive country dont need any amount of food, wile player will e confused "why it was 100 before?"

Reply:
Look again. When the colony is controlled by Varelses _or_ Primitives, it produces more than fifty. That was intentional.

Reply:
I do not see why it should produce different numbers